### PR TITLE
Refactor outbound calls

### PR DESCRIPTION
**Pull request by bytheway875**
Some refactors of outbound call controller logic based on feedback from @alaroia-cfa and @sniemeyer13. (See previously merged PR for more context: https://github.com/codeforamerica/vita-min/pull/680)

### CODE REVIEW

File: config/environments/development.rb [hunk 1]
```diff
@@ -73,5 +73,5 @@
   # Allow incoming connections over ngrok
   config.hosts << /[a-z0-9]+\.ngrok\.io/
   config.offseason = false
-  config.ngrok_url = "http://cec1d9b787e4.ngrok.io"
+  config.ngrok_url = "http://d0d376ec6d64.ngrok.io"
```
Comment (alaroia-cfa) [inline config/environments/development.rb:76]: Consider setting up `development_overrides.rb` as a file that's ignored in `.gitignore`. Not a blocker.

File: spec/factories/outbound_calls.rb [hunk 2]
```diff
@@ -23,7 +23,7 @@
   factory :outbound_call do
     client
     user
-    to_phone_number { "+18324658840" }
-    from_phone_number { "+18324651680" }
+    to_phone_number { "+15005550006" }
+    from_phone_number { "+18324650000" }
```
Comment (alaroia-cfa) [inline spec/factories/outbound_calls.rb:27]: :D

**Review by alaroia-cfa — APPROVED** [review]
Splendid! One comment; no blockers.

File: config/environments/development.rb (hunk 1 above)
Comment (bytheway875) [reply config/environments/development.rb:76]: Gah i meant to leave this out!

**Pull request merged**